css/main.css

```diff
@@ -84,22 +84,26 @@ kbd {
 	border: .4em solid red;
 }
 
+
 .topbar{
-	display: flex;
-	flex-wrap: nowrap;
-	justify-content: space-evenly;
-	align-items: stretch;
 	width: 100%;
-	max-height: 1.8em; /* 2*padding + text height + a teeny bit of padding*/
 	position: fixed;
 	background-color: var(--base-color);
-	padding: 0.3em;
-	box-sizing: border-box;
 	top: 0; /* hug the top of the window instead of having a buffer */
 	left: 0;
 	z-index: 1;/*Lets topbar render ontop of iframe*/
 }
 
+.topbarNav{
```

Comment:
something in here has broken the navbar's ability to stack its options when the browser width gets smaller so now you can't actually get to the guide and PRC tabs after a certain point.

Reply:
Ty for catching this. Will take a look.

Reply:
Fixed. Was because we have some narrow-screen-only formatting that I didn't change before.